examples/kubernetes/templates/v1/cilium-microk8s-ds.yaml.sed

```diff
@@ -0,0 +1,244 @@
+---
```

Comment:
@joestringer there's a containerd daemonset already available, can't we use that file instead?

Reply:
Done.